swayfmt/tests/macros.rs

```diff
@@ -0,0 +1,23 @@
+#[macro_export]
+macro_rules! assert_eq_pretty {
+    ($got:expr, $expected:expr) => {
+        let got = &*$got;
+        let expected = &*$expected;
```

Comment:
Do these need the extra deref? E.g. would the following be more flexible?
```suggestion
        let got = &$got;
        let expected = &$expected;
```

Reply:
Ah thanks for this catch! Makes sense since we're only passing in `&str`